Azure/azure-cli
# [CosmosDB] Add minimal TLS version as a parameter for create/update operations

**Related commands**

- az cosmosdb create
- az cosmosdb update

**Description**
This change allows for customers to specify the minimal TLS version while creating or updating an account. The accepted values for the minimal Tls version are 'Tls', 'Tls11', and 'Tls12', which correspond to the TLS versions 1.0, 1.1, and 1.2. If no TLS version is specified, the default Tls version is 1.0, except for Cassandra and Mongo API's, which only work with Tls version 1.2

**Testing Guide**
The following commands should create or update an account in which the minimal TLS version allowed is the one specified as 'TlsVersion': 

- az cosmosdb create -n DBName -g RGName --minimal-tls-version TlsVersion. 

- az cosmosdb update -n DBName -g RGName --minimal-tls-version TlsVersion. 

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
